portals/publisher/source/src/app/components/Apis/Details/TryOut/TryOutConsole.jsx

```diff
@@ -174,8 +205,19 @@ const TryOutConsole = () => {
             // If no deployment just show the OAS definition
             oasCopy = oasDefinition;
         }
+        if (oasCopy && api.advertiseInfo && api.advertiseInfo.advertised) {
```

Comment:
use the suggestion in APIconsole devportal

Reply:
Added with https://github.com/wso2/apim-apps/commit/5aac85c885fae9e259de37df89f53c6fc9b5d5dd